pyteal/__init__.py

```diff
@@ -8,6 +8,7 @@
     MIN_TEAL_VERSION,
     DEFAULT_TEAL_VERSION,
     MAX_PROGRAM_VERSION,
+    FRAME_POINTER_VERSION,
```

Comment:
I'd prefer not to export `FRAME_POINTER_VERSION` before we actually support frame pointer usage. Otherwise it seems misleading.

Reply:
removed from `__init__.py`